code/harness/DiskContents.h

```diff
@@ -17,55 +17,57 @@
 #include <vector>
 #include <map>
 
+using std::string;
```

Comment:
nit: we try not to have `using` statements in header files because it can make it hard to determine where namespaces are declared. Prefer `std::string` and such in header files rather than the `using std::string` statement